content/collections/experiment-apis/en/experiment-management-api-flags.md

```diff
@@ -573,7 +573,7 @@ Create a new variant for a flag
 |`key`| Required | string | The variant key. |
 |`description`| Optional | string | Description for the variant.|
 |`name`| Optional | string | Name for the variant.|
-|`payload`| Optional | string | Optional payload. Value must be a valid JSON element.|
+|`payload`| Optional | JSON | payload. Value must be a valid JSON element.|
```

Comment:
it already says it is "optional" so no need to say that again. But the description is super vague if you want to update that